src/test/regress/create_schedule

```diff
@@ -8,6 +8,7 @@ test: schemas_create
 test: views_create
 test: sequences_create
 test: index_create
+test: function_create
```

Comment:
I think the `create` piece is almost ready to go. But, one of the main goals of arbitrary configs is to also add a tests to non-create schedule such that they are executed from the worker nodes with all different kinds of configs.

For example, `SELECT warning(message) FROM warnings WHERE id = 1;` should go there and potentially few more executions. Like, a function does bunch of commands. 

I'd even consider getting HammerDB functions to this test, and have two copies each. The first copy is a distributed function/procedure, the other is non-distributed procedure. Both should produce the same output (and on pg as well).

If we do hammerDB, we might need to add `create_distributed_function` here as a no-op of Postgres schedule: https://github.com/citusdata/citus/blob/master/src/test/regress/sql/postgres.sql